playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerView.java

```diff
@@ -85,19 +105,25 @@ private Player.Listener getPlayerEventListener() {
             @Override
             public void onPlaybackStateChanged(int playbackState) {
                 switch (playbackState) {
+                    case Player.STATE_IDLE:
+                        if (shutterStaysOnRenderedFirstFrame && usingSpeedAdjustedRenderer) {
+                            if (shutterView != null) {
+                                shutterView.setVisibility(VISIBLE);
+                            }
+                        }
+                        break;
 
                     case Player.STATE_READY:
                         if (player != null && player.getPlayWhenReady()) {
                             log.d("ExoPlayerView READY. playWhenReady => true");
-                            if (shutterView != null) {
+                            if (shutterView != null && !(shutterStaysOnRenderedFirstFrame && usingSpeedAdjustedRenderer)) {
```

Comment:
this condition is now repeating 3 times in code `shutterView != null && !(shutterStaysOnRenderedFirstFrame && usingSpeedAdjustedRenderer)` so let's extract to a function